libs/core/src/components/sage-tab/sage-tab.tsx

```diff
@@ -0,0 +1,84 @@
+import { Component, Element, Host, h, Prop, Event, EventEmitter, State } from '@stencil/core';
+
+@Component({
+  tag: 'sage-tab',
+  styleUrl: 'sage-tab.scss',
+  shadow: false,
```

Comment:
Is it a best practice to explicitly set it to `false` or can this be omitted?

Reply:
For the subcomponents (`sage-tab` and `sage-tabpanel`), leaving the shadow dom open allows them to communicate with their parent (`sage-tabs`) while staying scoped to their parent shadowdom